include/nbl/asset/utils/CCompilerSet.h

```diff
@@ -15,8 +15,11 @@ namespace nbl::asset
 	{
 	public:
 		CCompilerSet(core::smart_refctd_ptr<system::ISystem>&& sys)
-			: m_HLSLCompiler(core::make_smart_refctd_ptr<CHLSLCompiler>(core::smart_refctd_ptr(sys)))
-			, m_GLSLCompiler(core::make_smart_refctd_ptr<CGLSLCompiler>(core::smart_refctd_ptr(sys)))
+			: 
+#ifdef _NBL_PLATFORM_WINDOWS_
+			m_HLSLCompiler(core::make_smart_refctd_ptr<CHLSLCompiler>(core::smart_refctd_ptr(sys))),
+#endif
```

Comment:
just going to disable this for Linux for now.

Reply:
good call